src/customer/web.ts

```diff
@@ -358,6 +352,40 @@ export default class Web extends WebAPI {
    * @param callback_uri - URI to send webhook when customer confirms identity
    */
   async requestEmailVerification(callback_uri: string): Promise<EmptyResponse> {
-	  return this.send("request_email_verification", { callback_uri });
+    return this.send("request_email_verification", { callback_uri });
+  }
+
+  /**
+   * Returns the dynamic configuration of a given group.
+   * @param opts - properties used to groups' filtering
+   */
+  async getDynamicConfiguration(opts: GetDynamicConfigurationRequest = {}): Promise<GetDynamicConfigurationResponse> {
+    return this.send("get_dynamic_configuration", opts);
+  }
+
+  /**
+   * Returns the configuration of a given group in a given version.
+   * @param group_id
+   * @param version
```

Comment:
do we have a description for that?

Reply:
Fixed. Completed the description for parameters.